dom.bs

```diff
@@ -483,10 +483,10 @@ empty list.
 {{TouchEvent}} interface and related interfaces. [[TOUCH-EVENTS]]
 
 <p>An <a>event</a> has an associated <dfn export for=Event>path</dfn>. A <a for=Event>path</a> is a
-<a for=/>list</a> of <a for=/>structs</a>. Each <a for=/>struct</a> consists of an
-<dfn for=Event/path>item</dfn> (an {{EventTarget}} object), an
-<dfn for=Event/path>item-in-shadow-tree</dfn> (a boolean), a <dfn for=Event/path>target</dfn> (a
-<a>potential event target</a>), a
+<a for=/>list</a> of <a for=/>structs</a>. Each <a for=/>struct</a> consists of a
+<dfn for=Event/path>current target</dfn> (an {{EventTarget}} object), a
+<dfn for=Event/path>current-target-in-shadow-tree</dfn> (a boolean), a
+<dfn for=Event/path>target</dfn> (a <a>potential event target</a>), a
```

Comment:
Maybe we can call this one *dispatch target* to further distinguish itself from other kinds of targets?

Reply:
Okay, so this is either the target passed into the dispatch algorithm or the target as adjusted for shadow tree encapsulation (some host). Does originating target reflect that well?

Reply:
Hm... maybe shadow-adjusted originating target? I think the key here is convey the semantics that the event looks as though it originated from that node in scripts.

Or we could call it *shadow adjusted target* or *visible originating target*. Or maybe even *embarkment target* or *source target*?